src/ptwt/_stationary_transform.py

```diff
@@ -18,18 +19,18 @@ def _swt(
     data: torch.Tensor,
     wavelet: Union[Wavelet, str],
     level: Optional[int] = None,
-    axis: Optional[int] = -1,
-) -> List[torch.Tensor]:
+    axis: int = -1,
+) -> list[torch.Tensor]:
```

Comment:
This has worked since Python 3.9 ( https://docs.python.org/3/whatsnew/3.9.html#type-hinting-generics-in-standard-collections ). We do not support 3.8, it is okay. Thanks for updating!